### DISCUSSION TITLE
FLOPS computing

### DISCUSSION THREAD

**Discussion by liziang297** [ultralytics/yolov5#3371] [Q&A]
I don't know why can't Model Summary show FLOPS when I run detect.py.
There are two pictures.
One is yolov5_master before April 11,another is yolov5_develop.
It used can show this item, but maybe there's something wrong after the update?
Thanks a lot!

![yolov5_develop](https://user-images.githubusercontent.com/80088752/119938740-722ad300-bfbf-11eb-9aef-6305b35acb65.png)
![yolov5_master](https://user-images.githubusercontent.com/80088752/119938771-7d7dfe80-bfbf-11eb-853b-1505fbbbd524.png)

**Comment by glenn-jocher** [comment]
@liziang297 FLOPS are automatically displayed if `thop` package is installed in your environment:
```
pip install thop
```

**Reply by edehino** [reply]
What does FLOPS @ 640 means in your Pretrained Checkpoints table? I understand that FLOPS is FLoating point Operations Per Second but I don't understand what @640 implies. @glenn-jocher

**Reply by glenn-jocher** [reply]
640 image size
